crates/bevy_diagnostic/src/system_information_diagnostics_plugin.rs

```diff
@@ -144,9 +158,21 @@ pub mod internal {
                 let used_mem = sys.used_memory() as f64 / BYTES_TO_GIB;
                 let current_used_mem = used_mem / total_mem * 100.0;
 
+                let process_mem_usage = sys
+                    .process(pid)
+                    .map(|p| p.memory() as f64 / 1024.0 / 1024.0) // Convert to MB
+                    .unwrap_or(0.0);
+
+                let process_cpu_usage = sys
+                    .process(pid)
+                    .map(|p| p.cpu_usage() as f64)
```

Comment:
The doc mentions that it should be divided by the number of core to get a more representative value.